tests/sysctl/test_sysctl.py

```diff
@@ -175,7 +177,7 @@ def test_proto_get_tuple(self) -> None:
 
     def test_proto_get_unknown(self) -> None:
         self.proto.lineReceived(b'net.unknown')
-        self.assertEqual(b'[error] net.unknown not found\n', self.tr.value())
+        self.assertEqual(b'[error] unknown not found\n', self.tr.value())
```

Comment:
Why aren't we presenting the whole path in the error message?

Reply:
Because now the error message comes directly from `ConnectionsManagerSysctl` and `SysctlProtocol` has no clue about the path because it is the of `SysctlRunner` now. Same bellow.

Reply:
I will proceed, but if you find this problematic I can open another PR.

Reply:
I guess we should fix the error message. Showing just the last part of the path is misleading. We can keep this conversation closed since I just started a new one.